cpp/cmake/Dependencies.cmake

```diff
@@ -182,6 +195,8 @@ if(BUILD_GTEST)
 	set(GTEST_BINARY_DIR ${PROJECT_BINARY_DIR}/googletest)
 	set(GTEST_INSTALL_DIR ${GTEST_BINARY_DIR}/install)
 	set(GTEST_LIB ${GTEST_INSTALL_DIR}/lib/libgtest_main.a)
+
+
```

Comment:
Stray lines? If they are intentional for clarity, I don't have an issue with them